docs/rfcs/0001_organizations.md

```diff
@@ -0,0 +1,98 @@
+# Organizations
+
+**Status:** Approved, waiting for implementation
+
+An organization encompasses a group of people working together to integrate many studies.
+
+## Why
+
+The concept of an an Organization arises from the need to manage groupings of studies and people with similar tooling. With an Organization, an institution or data coordinating center can establish a workspace for itself and administer its own studies and personnel. An Organization is intended to separate administrative needs from the data collection process.
+
+## How
+
+An Organization encompasses two important entities: studies and users. An Organization owns many studies where each study is related strictly to one Organization. An Organization also contains many users although each user may be a member of many Organizations.
+
+### Model
+
+#### Diagram
+![Relational Diagram](../_static/images/OrganizationModel.png)
+#### Schema
+These are fields to be set on the `Organization` model:
+```graphql
+type Organization {
+  id: ID!
+  name: String!
+  description: String
+  avatar: String
+  email: String
+  website: String
+}
+```
+
+### Behavior
+
+The following outlines use of Organizations as the user will experience them in the application.
+If a user logs in but is new or does not yet belong to any organizations, they will be prompted to create a new Organization or given guidance on getting access to an existing organization (see 'Joining').
+If a user belongs to one or more organizations, they will be presented with the dashboard for their last active organization.
+
+#### Creating
+Upon creation, the authenticated user who made the request to create the Organization will be added as an administrative member of the Organization.
+##### Schema
+```graphql
+type CreateOrganizationInput {
+  name: String!
+}
+createOrganization(input: CreateOrganizationInput): Organization!
+```
+##### Permissions
+
+Any authenticated user may create a new organization.
```

Comment:
Hmm maybe only superusers (I think that's what our current Admin group represents) should be able to create a new organization?

Reply:
Perhaps to start, but the idea is that anyone may join and create an organization.